From 4fe819b569b3f9d679766f7f2c27130b310d6ad4 Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Sun, 26 Apr 2020 01:43:32 +0900
Subject: [PATCH] reserv_mem_for_logo

reserv mem which can use for store some addition embed data
and we can get in by $fdtcontroladdr_end mem addr 

---
 common/board_f.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/board_f.c b/common/board_f.c
index 73254372..bbfc321c 100644
--- a/common/board_f.c
+++ b/common/board_f.c
@@ -557,1 +557,1 @@ 
-		gd->fdt_size = ALIGN(fdt_totalsize(gd->fdt_blob) + 0x1000, 32);
+		gd->fdt_size = ALIGN(fdt_totalsize(gd->fdt_blob) + 0x10000, 32);
 
-- 
2.17.1

